comp/api/api/apiimpl/internal/agent/agent.go

```diff
@@ -98,6 +98,7 @@ func SetupHandlers(
 		getWorkloadList(w, r, wmeta)
 	}).Methods("GET")
 	r.HandleFunc("/secrets", func(w http.ResponseWriter, r *http.Request) { secretInfo(w, r, secretResolver) }).Methods("GET")
+	r.HandleFunc("/secret/refresh", func(w http.ResponseWriter, r *http.Request) { secretRefresh(w, r, secretResolver) }).Methods("GET")
```

Comment:
```suggestion
	r.HandleFunc("/secret/refresh", func(w http.ResponseWriter, r *http.Request) { secretRefresh(w, r, secretRefresh) }).Methods("GET")
```

Reply:
This parameter should stay as `secretResolver`, that's the component that performs secret resolving. In contrast, `secretRefresh` is the function being called by the API endpoint.